Update xwidgets patch to a newer one

pcr910303 opened the pull request:
Fixes #94 👍

d12frosted commented:
Thank you very much. I will take a look at your PR very soon (tm).

d12frosted commented:
Thank you very much. I will take a look at your PR very soon (tm).

P. S. do you mind to push you patch to the patches folder of this repository? It's safer to have all patches in one place, so if you for some reason remove your gist, others are not affected.

pcr910303 commented:
@d12frosted I added the patch to my branch and modified the formula accordingly; the formula won't work until you merge this PR as the address specifies your master branch :-)

d12frosted commented on Formula/emacs-plus.rb:
```diff
@@ -153,8 +153,8 @@ class EmacsPlus < Formula
-      url "https://gist.githubusercontent.com/fuxialexander/0231e994fd27be6dd87db60339238813/raw/b30c2d3294835f41e2c8afa1e63571531a38f3cf/0_all_webkit.patch"
-      sha256 "f35b955aef31537d2ff163ec9bfcc2176dbcd0ea64f05440d98ec2988b82ce25"
+      url "https://raw.githubusercontent.com/d12frosted/homebrew-emacs-plus/master/patches/xwidgets_webkit_in_cocoa.patch"
+      sha256 "84bbff657c3823d5a671956a429a05a4f27accfc36b25d12ef14d6537df976ab"
```
I may be doing something wrong, but my hash is different. Actually the previous one:

```
λ curl https://raw.githubusercontent.com/d12frosted/homebrew-emacs-plus/581586bf896f266de8edb6cf9381a6285c690008/patches/xwidgets_webkit_in_cocoa.patch | sha256sum 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 79413  100 79413    0     0   324k      0 --:--:-- --:--:-- --:--:--  325k
f35b955aef31537d2ff163ec9bfcc2176dbcd0ea64f05440d98ec2988b82ce25  -
```

d12frosted commented:
OK, I am going to merge it and test 😸 hold tight!

d12frosted commented:
If the hash is not right, I'll just update it.

pcr910303 commented:
@d12frosted Sorry for that; I'm not sure what mixed up;;

```
$ curl https://raw.githubusercontent.com/d12frosted/homebrew-emacs-plus/master/patches/xwidgets_webkit_in_cocoa.patch | shasum -a 256
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 79413  100 79413    0     0  63123      0  0:00:01  0:00:01 --:--:-- 63126
f35b955aef31537d2ff163ec9bfcc2176dbcd0ea64f05440d98ec2988b82ce25  -
```

d12frosted commented:
NP, I will test it on my computer and will wait for brew to yell at me. Waiting for XCode to update :smile_cat:

d12frosted commented:
Ok, the update will take some time. Since xwidget was broken anyways, it's not critical if the hash is wrong.

pcr910303 commented:
@d12frosted Oops :-( The patch is already failing at HEAD  :-|
I'll try fixing the patch tonight and make another PR...

pcr910303 commented:
@d12frosted Oh, it looks like I uploaded the wrong patch :-(
Sorry for that...
